src/clients/save_network.rb

```diff
@@ -70,6 +77,7 @@ def adjust_for_network_disks(file)
       )
 
       log.info("Network based device: #{remote_access}")
+=end
```

Comment:
some time age we decided not to keep commented (disabled) code in sources. Is there any reason to keep disabled code?